core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -81,6 +86,34 @@ def parse_row(row):
     return [lsn, backup_label, tablespace]
 
 
+def convert_to_ns(data):
+    """
+    This function is used as a replacement when system doesn't have ns native support
+    in time and os function.
+    """
+    return int(data * NANOSECONDS_PER_SECOND)
+
+
+def current_time_ns():
+    """
+    unified function to return current time in ns
+    """
+    try:
+        return time.time_ns()
+    except AttributeError:
+        return convert_to_ns(time.time())
```

Comment:
`time_ns()` exists since python 3.7.  We should add a comment here explaining this, so that we later know that we can remove this.
See https://docs.python.org/3/library/time.html#time.time_ns

Reply:
Good catch. I really appreciate the way you improve (or force us) the code.
Done.